media/css/m24/components/navigation-refresh.scss

```diff
@@ -263,9 +273,26 @@
                 height: auto;
                 background-color: #fff;
                 overflow: hidden auto;
-                transition: 0.45s;
-                margin-top: 50px;
-                transform: translateX(100%);
+                margin-top: 98px;
+                animation: slide-in 0.45s ease;
```

Comment:
**issue:** this animation is applying on desktop viewports as well as mobile?

Reply:
This should be fixed now